src/interactive/repl.ts

```diff
@@ -113,6 +113,12 @@ async function startREPL(preserveFocus: boolean, showTerminal: boolean = true) {
     }
 }
 
+function killREPL() {
+    if (g_terminal) {
+        g_terminal.dispose()
```

Comment:
This will presumably then trigger some even handler that sets `g_terminal` to `undefined`, right?

Reply:
To `null`, but yes, [this one](https://github.com/julia-vscode/julia-vscode/blob/2d4d60dba1b3e74ad8742214c673d94aa0209a83/src/interactive/repl.ts#L507). Not sure if I shouldn't explicitly do that here though.

Reply:
That seems good enough, I think only having one point where this is set to `null` is actually nicer :)